scripts/ci/pre_commit/pre_commit_check_watcher_in_examples.py

```diff
@@ -59,6 +67,13 @@ def _check_file(file: Path):
                     f"        {WATCHER_APPEND_INSTRUCTION}\n\n"
                     "[yellow]as last instruction in your example DAG.[/]\n"
                 )
+        if PYTEST_FUNCTION not in content:
+            errors.append(
+                f"[yellow]The  example {file} missed the pytest function at the end.[/]\n\n"
+                "All example tests should have this function added:\n\n" + PYTEST_FUNCTION + "\n\n"
+                "[yellow]Automatically adding it now!\n"
+            )
+            file.write_text(content + "\n" + PYTEST_FUNCTION)
```

Comment:
Shouldn't it be preceded with 2 blank lines? Please check if the resulted content passes the black check.

Reply:
PYTEST_FUNCTION starts with "\n" :)